src/points.rkt

```diff
@@ -25,6 +26,10 @@
                   body ...)])
     (mk-pcontext pts* exs*)))
 
+(define (pcontext->lists context)
+  (for/lists (pts exs) ([(pt ex) (in-pcontext context)])
+    (values pt ex)))
```

Comment:
I think it's fine but why not just export `pcontext-points` and `pcontext-exacts`? Because they are vectors?